### PR TITLE
Shorten QC messages

### LINKED ISSUE #46: Shorten QC messages - remove repeated terms
Suggested enhancement: edit QC messages to remove repeated terms.  
For warnings:  
"WARNING: Low Coverage: Low coverage for all tiles (18.200 < 20 expected)" -> **change to:** "WARNING: Low coverage for all tiles (18.200 < 20 expected)"
For missing tiles:
"FAIL: Missing Tiles Error 1: 11.11% missing tiles; more than 5.00% missing tiles threshold. Low coverage depth (19.1 < 20.0 expected); you may need more WGS data." -> **change to:** "FAIL: 11.11% missing tiles > 5.00% QC setting. Low coverage depth (19.1 < 20.0 expected); you may need more WGS data."
For mixed samples:
"FAIL: Mixed Sample Error 2: Mixed subtype detected. Positive and negative tiles detected for the same target site "2417681" for subtype "1"."  -> **change to:** "FAIL: Mixed subtype detected. Positive and negative tiles detected for the same target site "2417681" for subtype "1"."

### PR DESCRIPTION
**Pull request by peterk87**
Fixes #46 by removing some redundancy in the QC messages. 

Signed-off-by: pkruczkiewicz <peter.kruczkiewicz@canada.ca>

### CODE REVIEW

File: bio_hansel/qc/checks.py [hunk 1]
```diff
@@ -132,18 +129,16 @@ def is_mixed_subtype(st: Subtype, df: DataFrame, *args) -> Tuple[Optional[str],
...
     if conflicting_tiles is None or conflicting_tiles.shape[0] == 0:
         return None, None
 
-    return QC.FAIL, '{}: Mixed subtype detected. Positive and negative tiles detected for ' \
-                    'the same target site "{}" for subtype "{}".'.format(
-                        QC.MIXED_SAMPLE_ERROR_2,
-                        '; '.join(conflicting_tiles['refposition'].astype(str).tolist()),
-                        st.subtype)
+    return QC.FAIL, ('Mixed subtype detected. Positive and negative tiles detected for '
+                     'the same target site "{}" for subtype "{}".').format(
```
Comment (gcttong) [inline bio_hansel/qc/checks.py]: Maybe make the message more clear for multiple target sites? "Positive and negative tiles detected for the following target sites: 282929, 228292, 2982822 for subtype 1, 2, 3" to make it easier for the user?

File: bio_hansel/qc/checks.py [hunk 2]
```diff
@@ -132,18 +129,16 @@ def is_mixed_subtype(st: Subtype, df: DataFrame, *args) -> Tuple[Optional[str],
     """
     if not st.are_subtypes_consistent:
         return QC.FAIL, \
-               '{}: Mixed subtypes found: "{}".'.format(
-                   QC.MIXED_SAMPLE_ERROR_2,
+               'Mixed subtypes found: "{}".'.format(
```
[the remark this answers is not in the document]
Comment (peterk87) [reply bio_hansel/qc/checks.py]: You're totally right. The language in the QC messages should be consistent. Thanks for this suggestion!

File: bio_hansel/qc/checks.py (hunk 1 above)
Comment (peterk87) [reply bio_hansel/qc/checks.py]: That QC message could definitely be clearer and there could definitely be multiple target sites where the positive and negative tiles are present. I'm wondering how to make this message more informative for non-experts to explain why it's not ideal to have both positive and negative tiles present at the same location.

**Comment by peterk87** [comment]
@gcttong I cleaned up the QC messages a bit in 4a81526468b761d56404d13c4bfcf31b78b623ab.

**Comment by gcttong** [comment]
ok alright, looks good!

**Pull request merged**